exercises/random_number.cpp

```diff
@@ -1,6 +1,10 @@
-/*
-  Write a program that generates a random number.
+#include <iostream>
+#include <time.h>
+#include <stdlib.h>
 
-  Output:
-  The random number is: 4
-*/
+using namespace std;
+
+int main(){
+	srand(time(NULL));
+	cout<<"The number is: "<<rand()<<"\n";
+}
```

Comment:
manca return 0, inoltre potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```